TransTest.hs

```diff
@@ -43,7 +43,8 @@ testDotGOutput = do
                        , "C1 C-"
                        , "C- C0"
                        , ".marking {A0 B1 C0}"
-                       , ".end"]
+                       , ".end"
+                       , ""]
 
```

Comment:
See my other comment. I think we should write invariants in comments, before the `.end` line.